### PR TITLE
"Uses mofo-dev" button

### LINKED ISSUE #10: let's make a button!
In the interest of getting links to this repo as visibly added to our other repos, it might be cool to make a little button graphic that we can add to README.md files that says something along the lines of "this repo conforms to superawesometown mofo guidelines" and links back here when clicked.

### PR DESCRIPTION
**Pull request by Pomax**
fixes #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<a href="https://github.com/MozillaFoundation/mofo-engineering-guide" style="text-decoration: none">
+<img src="https://cdn.rawgit.com/Pomax/mofo-engineering-guide/uses-mofo-dev/uses-mofo-dev.svg">
```
Comment (Pomax) [inline README.md]: note: we'll want to change this to the MozillaFoundation link post-merge

Comment (gvn) [inline README.md]: Do you want to do this with markdown instead?

File: README.md [hunk 2]
```diff
@@ -66,3 +70,13 @@ We use webhooks to trigger automatic deployments to staging environments from th
...
+
+## "Uses Mofo-dev" button
+
+Any project that follows our engineering guideline can be explicitly marked as doing so by adding the following button snippet to its README.md
+
+```
+<a href="https://github.com/MozillaFoundation/mofo-engineering-guide" style="text-decoration: none">
+<img src="https://cdn.rawgit.com/Pomax/mofo-engineering-guide/uses-mofo-dev/uses-mofo-dev.svg">
+</a>
+```
```
Comment (gvn) [inline README.md:78]: Can we do markdown instead?

**Comment by gvn** [comment]
Feel free to use `gh-pages` to host the SVG as well...

**Comment by Pomax** [comment]
oh hah, let me invalidate this PR and refile it against gh-pages instead. Much easier. (with the markdown version instead of html)

**Pull request closed without merging**